Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_installation.py

```diff
@@ -0,0 +1,251 @@
+import platform
+from os import path
+import json
+import subprocess
+import time
+import bpy
+import os
+
+EDITOR_COMMAND_ADD_SERVER = 1
+EDITOR_COMMAND_GET_PROJECT_PATH = 2
+
+DEBUG = 1
+
+from . import MeshSyncClientBlender as ms
+msb_context = ms.Context()
+
+unity_process = None
+
+def MS_MessageBox(message = "", title = "MeshSync", icon = 'INFO'):
+    def draw(self, context):
+        self.layout.label(text=message)
+    bpy.context.window_manager.popup_menu(draw, title = title, icon = icon)
+
+def msb_get_meshsync_entry():
+    if not DEBUG:
+        return "0.14.5-preview"
+
+    os = platform.system()
+    if os == 'Windows':
+        return "file:C:\\Users\\Sean Dillon\\MeshSync\\MeshSync~\\Packages\\com.unity.meshsync"
+    elif os == 'Darwin' or os == 'Linux':
+        return "file:/Users/sean.dillon/MeshSync/MeshSync~/Packages/com.unity.meshsync"
+
+    return None
+
+def msb_error_messages_for_status(status, context):
+    if status == 'SUCCESS':
+        return True
+    if status == 'INVALID_PATH':
+        MS_MessageBox("Path "+context.scene.meshsync_unity_project_path+" is not a Unity Project.")
+    elif status == 'EDITOR_NOT_EXISTS':
+        MS_MessageBox("Could not find Unity Editors in path "+ context.scene.meshsync_unity_editors_path+" .")
+    elif status == 'SERVER_NOT_ADDED':
+        MS_MessageBox("Could not add server to scene.")
+    elif status == 'LAUNCH FAILED':
+        MS_MessageBox("Could not launch project in path "+context.scene.meshsync_unity_project_path+" .")
+    elif status == 'SERVER_UNAVAILABLE':
+        MS_MessageBox("Could not connect with scene server.")
+
+    return False
+
+def msb_get_editor_path_suffix():
+    os = platform.system()
+    if os == 'Windows':
+        return path.join("Editor","Unity.exe")
+    elif os == 'Darwin':
+        return path.join("Unity.app","Contents","MacOS","Unity")
+    elif os == 'Linux':
+        return path.join("Editor","Unity")
```

Comment:
the doc about the location of the Editor is wrong https://docs.unity3d.com/Manual/EditorCommandLineArguments.html
I tested on Ubuntu, an its under Editor/Unity